lib/team_api/canonicalizer.rb

```diff
@@ -108,7 +110,7 @@ def self.hyphenate_yyyymmdd(timestamp)
     # different strings for the same thing.
     def self.canonicalize_tag_category(tags_xrefs)
       return if tags_xrefs.nil? || tags_xrefs.empty?
```

Comment:
how would you feel about `if tags_xrefs || tags_xrefs.any?` - I know `return` can be used to break, but I don't usually use `return` in my Ruby.